api/librdb-ext-api.h

```diff
@@ -93,29 +93,20 @@ _LIBRDB_API RdbxFilterKey *RDBX_createHandlersFilterKey(RdbParser *p,
 typedef struct RdbxToRespConf {
     /* TODO: support the option of expire, del, select db */
 
+    int delKeyBeforeWrite;
+    int applySelectDbCmds;  /* if not configured SELECT commands, all keys will be flatten into a single db */
+
     /* If supportRestore, then data-types will be translated to RESTORE with
      * raw data instead of data-types commands. This is a performance optimization
      * that requires to be version aligned. */
     int supportRestore;
 
-    /* TODO: support rdb2resp del key before write */
-    int delKeyBeforeWrite;
-    int skipAuxField;
-    int applySelectDbCmds;
-
-    /* relevant only if supportRestore is set. */
-    struct {
-        /* It is required to verify that the target (consumer) of the RESP payload
-         * is aligned with the version of the source RDB file. Otherwise, restore
-         * won't be respected */
-
-        /* Configure what is target RDB version. if equals 0, then the value
-         * will be resolved by the value of dstRedisVersion. */
-        int dstRdbVersion;
-
-        /* an alternative configuration to dstRdbVersion */
-        char *dstRedisVersion;
-    } restore;
+    /* dstRedisVersion is relevant only if supportRestore is set. It is required
+     * to verify that the version of Redis that will eventually will consume RESP
+     * output of this parser is aligned with the version of the source RDB file.
+     * Otherwise, RESTORE won't be honored and higher level commands will be
+     * generated by the parser instead */
+     const char *dstRedisVersion; /* "<major>.<minor>[.<patch>]" */
```

Comment:
so since we introduced this thing, we started using it for more than just the decision to send RESTORE or not.
i.e. for ABSTTL, and maybe others in the future.
so let's generalize this comment, and maybe other references to is (like the CLI help message)